omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -103,7 +103,28 @@
               {actions_vhost, <<"<%= @actions_vhost %>">>},
               {actions_exchange, <<"<%= @actions_exchange %>">>},
               {actions_fqdn, <<"<%= node['private_chef']['lb']['api_fqdn'] %>">>},
-
```

Comment:
should this include the queue monitor settings as well? `rabbitmq_queue_length_monitor_enabled` etc